docs/release-procedure.md

```diff
@@ -0,0 +1,15 @@
+# Release procedure
+
+First, resolve any potential merge conflicts with main on the develop branch
+
+1. `git checkout develop`
+2. `git pull origin develop`
+3. `git checkout main`
+4. `git merge develop` (there should be no merge conflicts)
+
+Then, draft a new release of the main branch on GitHub:
+
+1. Navigate to [https://github.com/WildMeOrg/codex-frontend/releases/new](https://github.com/WildMeOrg/codex-frontend/releases/new).
+2. Designate the target as the main branch.
+3. Create a new tag following the SemVer pattern outlined [here](https://semver.org/) (vX.X.X).
```

Comment:
make it more explicit that we need to have a string that looks exactly like vX.X.X - i know you say that here but i want to be very clear about it, since devOps services will be looking for that exact string

Reply:
also the tag should only be made after the PR merging develop into main has been approved and merged

Reply:
Updated in https://github.com/WildMeOrg/codex-frontend/pull/424/commits/5f52be6481f1d4a5568fa496ff399b5d68ad7346.